lumicks/pylake/force_calibration/tests/test_convenience.py

```diff
@@ -50,11 +50,13 @@ def test_active_force_calibration_active(reference_models):
     """This test tests merely whether values were handed to the low level API correctly, not whether
     the calibration results are correct."""
     data, f_sample = reference_models.lorentzian_td(4000, 1, 0.4, 14000, 78125)
+    oscillation = np.sin(77 * 2 * np.pi * np.arange(data.size) / 78125)
+    data += oscillation
```

Comment:
I'd suggest copy/pasting the commit message here as a comment since this is not immediately obvious.

Reply:
Good point. Will do.